docs/architecture/adr-062-collections-state-layer.md

```diff
@@ -78,7 +78,7 @@ These default implementations also offer safety around proper lexicographic orde
 
 Examples of the collections API can be found here:
 - introduction: https://github.com/NibiruChain/collections/tree/main/examples
-- usage in nibiru: [x/oracle](https://github.com/NibiruChain/nibiru/blob/master/x/oracle/keeper/keeper.go#L32), [x/perp](https://github.com/NibiruChain/nibiru/blob/main/x/perp/v2/keeper/keeper.go)
+- usage in nibiru: [x/oracle](https://github.com/NibiruChain/nibiru/blob/master/x/oracle/keeper/keeper.go#L32), [x/perp](https://github.com/NibiruChain/nibiru/blob/main/x/oracle/keeper/keeper.go)
```

Comment:
this was pointing to x/perp but now oracle. I don't think it is correct.

Reply:
good catch, I could use https://github.com/NibiruChain/nibiru/blob/main/x/epochs/keeper/keeper.go as its just examples of using collections. They no longer have perp as a module

Reply:
works! let's use maybe a commit instead of main; otherwise we may have to update this often.